src/constants/routing.ts

```diff
@@ -133,9 +135,15 @@ export const COMMON_BASES: ChainCurrencyList = {
     ExtendedEther.onChain(SupportedChainId.POLYGON),
     WRAPPED_NATIVE_CURRENCY[SupportedChainId.POLYGON],
   ],
+  [SupportedChainId.POLYGON]: [
+    ExtendedEther.onChain(SupportedChainId.POLYGON),
```

Comment:
nit. also defined at 135, small perf improvement by defining once.. not a big dal